### ISSUE TITLE
Ensure CA certificates are updated prior to extraction from Debian

### ISSUE THREAD

**Issue by chadlwilson** [GoogleContainerTools/distroless#753]
Currently cacerts appear to be extracted from the relevant Debian package but they are not updated using `update-ca-certificates` prior to their being included/parsed/extracted for use in the various distroless base variants.

There is a [TODO noting this](https://github.com/GoogleContainerTools/distroless/blob/063a079c1a87bad3369cb9daf05e371e925c0c91/base/base.bzl#L31), however I couldn't find an open issue that seeks to note this weakness and address it.

Currently this seems to mean a number of more recent root CAs are not trusted by default when using with dependent tools, e.g changes such [as this one](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/NSS_3.54_release_notes) that add new CAs, but also remove ones that should no longer be trusted.

Putting aside for now whether the root of trust should be baked into an image or not (perhaps init-containered/mounted over the top at runtime), the CA certificates are currently baked in, so it seems appropriate that they should be regularly updated on rebuild, according to the standard Debian practice.

**Comment by michaelbeaumont** [comment]
@loosebazooka in what sense is/where was this completed? I still see https://github.com/GoogleContainerTools/distroless/blob/2bbea5c96b7d53dfd00cf294509a1ed7f5e41717/static/static.bzl#L44

**Comment by loosebazooka** [comment]
huh? IWe've been handling this by consuming the package directly from debian and then running a custom script on top of it. Are you running into something?

**Comment by michaelbeaumont** [comment]
Nope, I'm just looking at a workaround for this issue, wondering if it's still necessary and the TODO linked in the original comment is still there so 🤷

**Comment by loosebazooka** [comment]
so `update-ca-certificates` and the `cacerts` rule differ in a couple ways:
1. cacerts doesn't create the openssl hashed symlinks in /etc/ssl/certs, instead forcing openssl to use the env var: SSL_CERT_FILE
1. No custom certs or cert filtering mostly because we run this at the very root of the distroless build

**Comment by loosebazooka** [comment]
But the issue itself about "updated" is handled by keeping the deb package up to date.
